### DISCUSSION TITLE
Why when i create new project it creates under flixel-tools

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2478] [Q&A]
I type the usual command 
![image](https://user-images.githubusercontent.com/60806448/148037775-3ed557e3-04df-4ac1-b2c3-ced6b16d30db.png)
to create new directory for project
And for some reason it creates under flixel-tools
If i move it on D:\ upon building it tells me "the system cannot find the path specified"
Same thing i get upon building it in flixel-tools, but it actually compiles properly, just doesn't launch on it's own (i still can launch it manually, through windows explorer)

Why does that happen, and can I fix it somehow?

**Comment by RichardBray** [comment]
I personally don't use flixel-tools but from what I can see it is the command-line interface of flixel https://haxeflixel.com/documentation/flixel-tools/. 

So when you do `flixel tpl` you're using flixel-tools so that might be the correct behaviour 🤷 
But I'm not sure. Maybe try using the flixel jam template if you don't want to use the CLI https://github.com/HaxeFlixel/game-jam-template

**Comment by Geokureli** [comment]
flixel tpl creates the project wherever you're terminal was pointing when you ran it. you need to `cd ../../some/other/folder` Not sure Why D: doesn't work, tho

**Reply by NikRetaNCAM** [reply]
> flixel tpl creates the project wherever you're terminal was pointing when you ran it. you need to `cd ../../some/other/folder` Not sure Why D: doesn't work, tho

ohhh
i see
also does that have anything to do with the game not launching by itself after compiling?

**Reply by Geokureli** [reply]
are you compiling via `lime build`, or `lime test`, build just compiles, `lime run [target]` will launch it, lime test will do both

**Reply by NikRetaNCAM** [reply]
> are you compiling via `lime build`, or `lime test`, build just compiles, `lime run [target]` will launch it, lime test will do both

im always using  lime test

but it gives me "system cannot find the path specified" (practically it works for me the same as lime build)
